framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java

```diff
@@ -348,16 +349,38 @@ private static int getMissingCount(Map<ColumnList, Integer> map) {
   }
 
   private static int compareTo(ColumnList list1, ColumnList list2,
-      Map<Integer, String> columnIndexAndOrder, int start) {
```

Comment:
The new method signatures use lists instead of maps to support duplicate column names when json string fields are used in the order-by clause.  The code that is added to preserve the original method signatures mostly convert the original maps into lists, and vice versa.